www/js/app.js

```diff
@@ -1132,6 +1171,8 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 $("#searchingArticles").hide();
                 alert("Article with title " + title + " not found in the archive");
             } else {
+                params.isLandingPage = false;
+                $('#activeContent').alert('close');
```

Comment:
Just for my understanding : what is this .alert function?

Reply:
@mossroy This is the standard way of closing a Bootstrap alert. See https://getbootstrap.com/docs/4.0/components/alerts/#methods . However, in one instance it caused an exception in both FF and in IE11, and I think it was because Bootstrap hadn't finished loading by the time it was called. So, you will see one alternative in the code which is simply a jQuery .hide() command. That doesn't actually dismiss the alert, just hides it, whereas this .alert function does.

Reply:
OK, thanks for the pointer